src/EA.Weee.Web/Areas/AatfReturn/Controllers/ObligatedSentOnController.cs

```diff
@@ -50,7 +50,8 @@ public virtual async Task<ActionResult> Index(Guid returnId, Guid organisationId
                     ReturnData = @return,
                     AatfId = aatfId,
                     OperatorName = operatorName,
-                    WeeeSentOnId = weeeSentOnId
+                    WeeeSentOnId = weeeSentOnId,
+                    PastedData = TempData["pastedValues"] as ObligatedCategoryValue
```

Comment:
Can we add a unit test to make sure that if the pasted values are provided they are set on the mapper object? ta...

Reply:
Just added a test.